README.md

```diff
@@ -4,133 +4,64 @@
-3. Run `make deps/up` to startup service dependencies
-4. Run `make migrate` to migrate database with the most updated schema
-
-### Management API Flavor
-To start the management-api flavor locally, run:
-```
-make run/management-api
-```
-
-To test if the service (with dependencies) is up and running, try to create a scheduler by running the following command:
-```
-curl --location --request POST "localhost:8080/schedulers" \
---header "Content-Type: application/json" \
---header "Accept: application/json" \
---data-raw "{
-	\"name\": \"scheduler-test\",
-	\"game\": \"game-test\",
-	\"state\": \"creating\",
-	\"portRange\": {
-		\"start\": 1,
-		\"end\": 1000
-	},
-	\"maxSurge\": \"10%\",
-	\"spec\": {
-		\"terminationGracePeriod\": \"100\",
-		\"containers\": [
-			{
-				\"name\": \"alpine\",
-				\"image\": \"alpine\",
-				\"imagePullPolicy\": \"IfNotPresent\",
-				\"command\": [ \"sh\", \"-c\", \"while true; do sleep 1; done\" ],
-				\"environment\": [
-					{
-						\"name\": \"env-var-name\",
-						\"value\": \"env-var-value\"
-					}
-				],
-				\"requests\": {
-					\"memory\": \"100Mi\",
-					\"cpu\": \"100m\"
-				},
-				\"limits\": {
-					\"memory\": \"200Mi\",
-					\"cpu\": \"200m\"
-				},
-				\"ports\": [
-					{
-						\"name\": \"port-name\",
-						\"protocol\": \"tcp\",
-						\"port\": 12345,
-						\"hostPort\": 54321
-					}
-				]
-			}
-		],
-		\"toleration\": \"\",
-		\"affinity\": \"\"
-	},
-	\"forwarders\": [
-		{
-            \"name\": \"test-fwd\",
-            \"enable\": true,
-            \"type\": \"gRPC\",
-            \"address\": \"www.NON-EXISTANT-URL.com\",
-            \"options\": {
-               \"timeout\": 1000,
-               \"metadata\": {}
-            }
-		}
-    ]
-}"'
-```
-
-### Worker Flavor
-To start the worker flavor locally, run:
-```
-make run/worker
-```
-
-If you've create a scheduler following the last steps of `management-api` test, starting the worker will execute the `create_scheduler` operation, you can check if the operation is executed by executing the following command:
-```
-export KUBECONFIG=$(pwd)/kubeconfig.yaml
-kubectl get namespaces | grep zooba
-```
-
-### Runtime watcher flavor
-To start the runtime watcher flavor locally, run:
-```
-make run/runtime-watcher
-```
-
-## Running tests
-
-1. Run `make run/unit-tests` to run all unit tests
-2. Run `make run/integration-tests` to run all integration tests
-3. Run `make run/e2e-tests` to run all E2E tests. NOTE: Currently it is not
-   possible to run it with the development environment set. This command will
-   stop the dev dependencies before running.
-4. Run `make lint` to run all registered linters
+[//]: # (List all the modules or other dependencies that module has, if possible with links to their repositories.)
+
+Maestro does not have any dependencies, but it provides an events forwarding feature that can be used to integrate with external matchmaking services.
```

Comment:
On the `v9` documentation I wrote that Maestro does not depend on any other system, only the internal databases and the runtime system where it resides